src/test/java/org/helpberkeley/memberdata/MainTest.java

```diff
@@ -866,6 +866,28 @@ public void updateMemberDataNoUpdatesTest() throws IOException, CsvException {
         assertThat(WorkRequestHandler.getLastStatusPost().raw).contains(MessageFormat.format(Main.UPDATE_USERS_NO_UPDATES, "update-member-data-no-updates.csv"));
     }
 
+    @Test
+    public void updateMemberDataOverrideMemberLimitSucceed() throws IOException, CsvException {
+        String request = readResourceFile("last-replies-data-request-template-extra.json")
+                .replace("REPLACE_DATE", yesterday())
+                .replace("REPLACE_EXTRA", "disable size audit")
+                .replaceAll("REPLACE_FILENAME","update-member-data-multiple-updates.csv");
+        HttpClientSimulator.setQueryResponseData(
+                Constants.QUERY_GET_REQUESTS_LAST_REPLIES, request);
+        String usersFile = findFile(Constants.MEMBERDATA_RAW_FILE, "csv");
+        String[] args = { Options.COMMAND_WORK_REQUESTS, usersFile };
+        try {
+            WorkflowExporter.setMemberLimit(2);
+            Main.main(args);
+            assertThat(WorkflowExporter.getMemberLimit() == 15); //size of user list in users.json
+        } finally {
+            WorkflowExporter.setMemberLimit(WorkflowExporter.DEFAULT_MEMBER_LIMIT);
```

Comment:
Now that we've made this static we need this same finally treatment in every test that uses setMemberLimit().